api/src/services/script-runner.service.ts

```diff
@@ -603,15 +603,39 @@ export class ScriptRunnerService {
           WHERE jp.programs_id = '${prog.id}'
         `);
 
+      //obj with program title as key and updated program description as value
+      const updatedProgInfo = {
+        Families:
+          "This property offers housing for all family types, with no age restrictions (other than the lead applicant must be over 18 years old). Contact the property manager for more information. Please select ''I am a part of this community'' to continue the application.",
+        'Supportive Housing for the Homeless':
+          'This property offers housing for those experiencing homelessness, and may require additional processes that applicants must complete to qualify.',
+        Veterans:
+          'This property offers housing for those who have served active duty in branches of the U.S. Armed Forces, and were discharged under conditions other than dishonorable.',
+      };
+      const description = Object.keys(updatedProgInfo).includes(prog.title)
```

Comment:
nit: You could simplify this by doing something like this
```
      const description = updatedProgInfo[prog.title] || prog.description;
```

Reply:
Genius